bower.json

```diff
@@ -19,7 +19,7 @@
     "**/.*"
   ],
   "dependencies": {
-    "angular": "~1.2.16",
+    "angular": "~1.5.5",
```

Comment:
Thank you for sending a pull request!
I think it's better to use ">=1.2.0 <2.0". How about you?